docs/glossary_concepts.txt

```diff
@@ -0,0 +1,177 @@
+* **canvas board** TBD
+* **cast vote record (CVR)** - An electronic record
+* **chain-of-custody** TBD
+* **choice** TBD candidate or YES/NO option
+* **closeout period** - The period beginning on the earlier of 90 days
+* **closeout plan** TBD
+* **comma separated value** (aka CSV)
+* **compliance audit** - An audit which checks that the audit trail is
+* **contest margin** TBD
+* **contest name** - The title of a contest.
+* **contest** - A partisan or nonpartisan candidate race, or a
+* **continuous integration** TBD
+* **coordinated election** - Coordinated Elections occur on the first
+* **county administrator** - The designated representative(s) of each
+* **county auditing interface** TBD
+* **county clerk** TBD
+* **county dashboard** TBD
+* **county name** TBD
+* **county number** as part of the ballot manifest
+* **county** TBD
+* **credentials** TBD
+* **damage** TBD
+* **dashboard** TBD
+* **data synchronization** TBD
+* **database table** TBD
+* **deliverable** - Any tangible or intangible object produced by
+* **design-by-contract** TBD
+* **detailed project plan** TBD
+* **developer dashboard** TBD
+* **digital ballot adjudication** TBD
+* **disaster** - An event that makes it impossible for Contractor to
+* **discrepency** TBD
+* **domain engineering** TBD
+* **domain model** TBD
+* **duplicated ballot** TBD
+* **election canvass** TBD
+* **election day** - The final day on which voters can cast a ballot in a
+* **electoral system** TBD
+* **email** TBD
+* **equipment** TBD
+* **evidence-based elections** - An approach to achieving election integrity
+* **fault tolerance** TBD
+* **formal method** TBD
+* **formal model** TBD
+* **full hand count** - TBD along these lines: A procedure for
+* **git** TBD
+* **go live date** TBD
+* **hash function** TBD, mentioning specifically SHA-256
+* **imprinted ballot** - A ballot on which a unique ballot identifier has
+* **issue tracker** TBD
+* **key personnel** - The position or positions that are specifically
+* **late-tabulation ballot** - A ballot which is tabulated after the CVR
+* **list of ballots to audit** TBD
+* **load testing** TBD
+* **mail ballot** TBD
+* **mandatory contest** - A contest which is subject to a risk limit and is
+* **manual ballot adjudication** TBD
+* **margin** TBD
+* **non-voter-verifiable ballot** (NVVB) - A ballot for which there is
+* **offeror** - A vendor that submits a responsible bid for this
+* **operational start date** - When the State authorizes Contractor to
+* **opportunistic audit** - An auditing technique designed to efficiently
+* **opportunistic contest** - A contest to be audited opportunistically.
+* **original ballot** TBD
+* **other personnel** - Individuals and Subcontractors, in addition to Key
+* **overstatement of the margin** TBD
+* **overvote** TBD
+* **paper ballot** TBD
+* **password** TBD
+* **penetration testing** TBD
+* **phantom ballot** A ballot which is listed in the manifest, but is
```

Comment:
It is a ballot which cannot be located. The current glossary definition uses it for both counted an uncounted ballots, which we might want to separate out, though the treatment would be the same.